pkgs/development/python-modules/pytest-playwright/default.nix

```
@@ -14,7 +14,7 @@
 
 buildPythonPackage rec {
   pname = "pytest-playwright";
-  version = "0.3.3";
+  version = "0.4.2";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
```

```suggestion
  disabled = pythonOlder "3.8";
```
https://github.com/microsoft/playwright-pytest/pull/177